README.rst

```diff
@@ -39,3 +39,16 @@ See help from python module:
     The generated tj3-file, can at the moment not be parsed by TaskJuggler directly. Only the tasks are exported
     to the tj3-file. The list of tasks needs to be embedded in a complete tj3-file. See the
     `TaskJuggler website <http://taskjuggler.org/>`_ for more details.
+
+.. note::
+
+    Unresolved tasks with logged time, i.e. time spent, will have their 'start' property set to the set current date
+    and time minus the logged time, calculated with 8 hours per workday and a default of 5 allocated workdays per week
+    with the day(s) off ending on Sunday. The latter number can be changed.
```

Comment:
Shouldn't we also explain that resolved tasks are considered too? Or this case is handled by the Jira query in the makefile and we don't want to talk about that here?

Reply:
In my opinion there should have been a warning about the fact that resolved tasks were not supported. This PR would then remove that warning rather than advertise support. 

I've added this note, which adds some value: eaa2fa1e39d31bae8f0a5512235d8e84320b10bc

Reply:
Thanks